resources/windows/wix/project.wxs.erb

```diff
@@ -55,16 +55,22 @@
     <!-- INSTALLDIR -->
     <CustomAction Id="SaveCmdLineInstallDir" Property="CMDLINE_INSTALLDIR" Value="[INSTALLDIR]" Execute="firstSequence" />
     <CustomAction Id="SetFromCmdLineInstallDir" Property="INSTALLDIR" Value="[CMDLINE_INSTALLDIR]" Execute="firstSequence" />
+    <%= File.read("resources/windows/CustomActions") if File.exist?("resources/windows/CustomActions") -%>
```

Comment:
NP - will put a concept PR together based on discussions we had with @ferventcoder earlier in week.